algebra-utils/src/msm/variable_base.rs

```diff
@@ -0,0 +1,603 @@
+use algebra::{AffineCurve, BigInteger, Field, FpParameters, PrimeField, ProjectiveCurve};
```

Comment:
I would suggest to eliminate MSM_SD as it's not working properly

Reply:
Done